cloud-data-protection/src/components/register.tsx

```diff
@@ -0,0 +1,71 @@
+import React, {useState} from "react";
+import './register.css';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/olivierbouchomsfontys/cloud-data-protection/pullRequest?prid=7219402)
```suggestion
import "./register.css";
```